rs/registry/canister/canister/registry.did

```diff
@@ -132,12 +132,16 @@ type KeyConfig = record {
   max_queue_size : opt nat32;
 };
 
-type MasterPublicKeyId = variant { Schnorr : SchnorrKeyId; Ecdsa : EcdsaKeyId };
+type MasterPublicKeyId = variant { Schnorr : SchnorrKeyId; Ecdsa : EcdsaKeyId; VetKd : VetKdKeyId };
 
 type SchnorrKeyId = record { algorithm : SchnorrAlgorithm; name : text };
 
 type SchnorrAlgorithm = variant { ed25519; bip340secp256k1 };
 
+type VetKdKeyId = record { curve: VetKdCurve; name: text };
+
+type VetKdCurve = variant { bls12_381_g2 };
```

Comment:
Wondering if it's worth having the underscores in this variant name. This leads to an even larger number of possible spellings for the same concept, as e.g., in camel case you drop the underscores.

Reply:
I created a dedicated ticket (CRP-2635) for clarifying the naming and variant spelling with the IC spec owners and acquire approval of the management canister API PR (https://github.com/dfinity/interface-spec/pull/158). With this, I'm resolving the this conversation here.